Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -212,6 +213,16 @@ class BrowserViewController: UIViewController {
         contentBlockListDeferred = ContentBlockerHelper.compileBundledLists()
         
         setupRewardsObservers()
+        
+        if let rewards = rewards {
+            notificationsHandler = AdsNotificationHandler(ads: rewards.ads, presentingController: self)
```

Comment:
I wonder: will it work well when for example settings or bookmarks panel are open? or AlertViewController?

Reply:
great stuff, hard to find a flaw in your PRs recently :D
pr is approved